### PR TITLE
feat(features/introductionsection): 81 add introduction section in fe…

### LINKED ISSUE #81: Introduction Section
Related to #78 

Image

### PR DESCRIPTION
**Pull request by Abdullah-Al-Numan**
feature

used tailwindcss, create introduction section in feature

# Description

import checkmark icon from element/icon/checkmark.

Fixes #81 

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
introduction

Closes #81

### CODE REVIEW

File: apps/akij-food-beverage/src/components/ui/features/IntroductionSection/Introduction.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+import Introductions from "./Introduction";
+
+export default {
+  component: Introductions,
+  title: "Features/Introduction",
+  parameters: { layout: "fullscreen" },
+} as Meta;
+
+export const Introduction : Story = () => <Introductions />
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IntroductionSection/Introduction.stories.tsx:11]: Add a new line

File: apps/akij-food-beverage/src/components/ui/features/IntroductionSection/useFetch.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     setState({...state, currentIndex: newcurrentIndex})
+  }
+  const prevProduct = () =>{
+    const newcurrentIndex = currentIndex-1 < 0 ? products.length -1 : currentIndex-1;
+    setState({...state, currentIndex: newcurrentIndex})
+  }    
+  const { loading, error, products, currentIndex } = state;
+  return {loading, error, products, currentIndex,nextProduct,prevProduct};
+}
+export default useFetch;
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IntroductionSection/useFetch.ts:33]: Add a new line

File: apps/akij-food-beverage/src/styles/globals.css [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+    .brand{
+        @apply text-transparent bg-clip-text bg-gradient-to-r from-blue-400 to-blue-600;
+    }
+    .description{
+        @apply my-8 leading-7 tracking-wide text-slate-700;
+    }
+    .customflex{
+        @apply flex items-center space-x-2;
+    }
+  }
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/styles/globals.css:26]: Add a new line

File: packages/ui/components/elements/Icon/arrow.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const Arrow = () => {
+  return (
+    <svg className="transition ease-in-out delay-150 hover:scale-110 duration-200  hover:fill-blue-500" fill="#000000" height="31px" width="32px" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 330 330">
+    <path id="XMLID_27_" d="M15,180h263.787l-49.394,49.394c-5.858,5.857-5.858,15.355,0,21.213C232.322,253.535,236.161,255,240,255
+        s7.678-1.465,10.606-4.394l75-75c5.858-5.857,5.858-15.355,0-21.213l-75-75c-5.857-5.857-15.355-5.857-21.213,0
+        c-5.858,5.857-5.858,15.355,0,21.213L278.787,150H15c-8.284,0-15,6.716-15,15S6.716,180,15,180z"/>
+    </svg>  )
+}
+
+export default Arrow
```
Comment (TanvirAlam) [inline packages/ui/components/elements/Icon/arrow.tsx:10]: Add a new line

File: apps/akij-food-beverage/src/components/ui/features/IntroductionSection/useFetch.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+              const products = await fetchProducts()
+              setState({ products, loading: false, error: '',currentIndex: 0 })
+            } catch (e) {
+              setState({ products, loading: false, error: e.message, currentIndex: 0 })
+            }
+          }
+          fetchData()
+      }, [])
+  const nextProduct = () =>{
+     const newcurrentIndex = (state.currentIndex+1) % state.products.length;
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IntroductionSection/useFetch.ts:23]: clear your Camal caes: `newCurrentIndex`

File: apps/akij-food-beverage/src/components/ui/features/IntroductionSection/useFetch.ts [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+     setState({...state, currentIndex: newcurrentIndex})
+  }
+  const prevProduct = () =>{
+    const newcurrentIndex = currentIndex-1 < 0 ? products.length -1 : currentIndex-1;
+    setState({...state, currentIndex: newcurrentIndex})
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IntroductionSection/useFetch.ts:28]: clear your Camal caes: newCurrentIndex

File: apps/akij-food-beverage/src/components/ui/features/IntroductionSection/Introduction.tsx [hunk 7]
```diff
@@ -0,0 +1,55 @@
+import CheckmarkIcon from "ui/components/elements/Icon/checkmark";
+import Arrow from "ui/components/elements/Icon/arrow";
+import { urlForThumbnail } from './../../../../utils/imageProcess';
+import { Alert, CircularProgress} from '@mui/material';
+import useFetch from "./useFetch";
+const Introduction = () => {
+  const {products,loading,error,currentIndex,nextProduct,prevProduct} = useFetch();
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IntroductionSection/Introduction.tsx:7]: why didnt you put this in `utils` folder?

File: apps/akij-food-beverage/src/pages/index.tsx [hunk 8]
```diff
@@ -3,6 +3,7 @@ import { Layout } from '~/components/layouts'
 import { Footer } from '../components/ui/features/Footer'
 import { Header } from '../components/ui/features/Header'
 import { Hero } from '../components/ui/features/Hero'
+import IntroductionSection  from '~/components/ui/features/IntroductionSection/Introduction'
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/pages/index.tsx:6]: add `index` in you folder, so you dont have to so deep

**Review by TanvirAlam — CHANGES_REQUESTED** [review]
looks nice @Abdullah-Al-Numan

File: apps/akij-food-beverage/src/components/ui/features/IntroductionSection/useFetch.ts (hunk 2 above)
Comment (Abdullah-Al-Numan) [reply apps/akij-food-beverage/src/components/ui/features/IntroductionSection/useFetch.ts:33]: Done

File: apps/akij-food-beverage/src/styles/globals.css (hunk 3 above)
Comment (Abdullah-Al-Numan) [reply apps/akij-food-beverage/src/styles/globals.css:26]: Done

File: packages/ui/components/elements/Icon/arrow.tsx (hunk 4 above)
Comment (Abdullah-Al-Numan) [reply packages/ui/components/elements/Icon/arrow.tsx:10]: Done

File: apps/akij-food-beverage/src/components/ui/features/IntroductionSection/useFetch.ts (hunk 5 above)
Comment (Abdullah-Al-Numan) [reply apps/akij-food-beverage/src/components/ui/features/IntroductionSection/useFetch.ts:23]: Done

File: apps/akij-food-beverage/src/components/ui/features/IntroductionSection/useFetch.ts (hunk 6 above)
Comment (Abdullah-Al-Numan) [reply apps/akij-food-beverage/src/components/ui/features/IntroductionSection/useFetch.ts:28]: Done

File: apps/akij-food-beverage/src/components/ui/features/IntroductionSection/Introduction.tsx (hunk 7 above)
Comment (Abdullah-Al-Numan) [reply apps/akij-food-beverage/src/components/ui/features/IntroductionSection/Introduction.tsx:7]: Done

File: apps/akij-food-beverage/src/pages/index.tsx (hunk 8 above)
Comment (Abdullah-Al-Numan) [reply apps/akij-food-beverage/src/pages/index.tsx:6]: Done

File: apps/akij-food-beverage/src/components/ui/features/IntroductionSection/Introduction.stories.tsx (hunk 1 above)
Comment (Abdullah-Al-Numan) [reply apps/akij-food-beverage/src/components/ui/features/IntroductionSection/Introduction.stories.tsx]: Done

**Review by TanvirAlam — APPROVED** [review]
LGTM :+1:

**Pull request merged**